src/codeview.js

```diff
@@ -135,6 +142,9 @@ var Codeview = GObject.registerClass({
     Properties: {
         text: GObject.ParamSpec.string('text', 'Text', 'Source code being edited',
             GObject.ParamFlags.READWRITE, ''),
+        errors: GObject.ParamSpec.uint('errors', 'Errors',
```

Comment:
There is already a `this._numErrors` attribute. You should use that or rename it.